### PR TITLE
🐎  Fixed With/WithoutBurst usage.

### PR DESCRIPTION
**Pull request by nicolasgramlich**

### CODE REVIEW

File: Runtime/Systems/SkillSystems.cs [hunk 1]
```diff
@@ -53,8 +50,7 @@ protected override void OnUpdate()
                     }
                     sbArray[i] = Skill;
                 }
-            }).WithBurst()
```
Comment (nicolasgramlich) [inline Runtime/Systems/SkillSystems.cs:56]: `WithBurst` is the default, unless you're trying to be explicit here.

File: Runtime/Systems/SkillSystems.cs [hunk 2]
```diff
@@ -30,15 +30,12 @@ protected override void OnUpdate()
                     }
                     sbArray[i] = Skill;
                 }
-            }).WithoutBurst()
```
Comment (nicolasgramlich) [inline Runtime/Systems/SkillSystems.cs:33]: This should be burst compatible.

Comment (WaynGames) [reply Runtime/Systems/SkillSystems.cs:33]: It is.

File: Runtime/Systems/SkillSystems.cs (hunk 1 above)
Comment (WaynGames) [reply Runtime/Systems/SkillSystems.cs:56]: Yes I try to be as explicit as possible. Like for the dependancy management.

**Comment by WaynGames** [comment]
Won't merge, fixed the "without burst" in the SkillUpdateSystem in the master branch.

**Pull request closed without merging**